CHANGES.rst

```diff
@@ -4,7 +4,9 @@
 4.1.3 (unreleased)
 ------------------
 
-- TBD
+- Fix _p_changed when removing items from small pure-Python
+  BTrees/TreeSets and when adding to Sets. See:
+  https://github.com/zopefoundation/BTrees/issues/13
```

Comment:
Don't we also re-fix #11?  Should be mentioned here (so that folks who experienced the bug in 4.1.2 know to expect that it is fixed).